core/field_textinput.js

```diff
@@ -207,15 +207,15 @@ Blockly.FieldTextInput.prototype.bindEvents_ = function(htmlInput) {
   // Bind to keydown -- trap Enter without IME and Esc to hide.
   htmlInput.onKeyDownWrapper_ =
       Blockly.bindEventWithChecks_(htmlInput, 'keydown', this,
-      this.onHtmlInputKeyDown_);
+          this.onHtmlInputKeyDown_);
```

Comment:
Not objecting, but I wonder about this.  What are the rules for requiring a second (third, fourth?) 4 space indent?

Reply:
I also wonder.  Maybe there's a way to turn off checking after too much nesting.